src/Connect.js

```diff
@@ -107,10 +117,20 @@ class Connect {
   }
 
   request ({uri, topic, uriHandler}) {
-    this.isOnMobile
-      ? this.mobileUriHandler(uri)
-      : (uriHandler || this.uriHandler)(uri, topic.cancel)
-    if (this.closeUriHandler) {
+    if (this.pushToken) this.credentials.push(this.pushToken, {url: uri})
```

Comment:
@zachferland this should `return` or the other uriHandlers are called later down